docs/setup_guides/hpc.rst

```diff
@@ -170,15 +189,15 @@ In our case, the Cheyenne super computer uses the PBS job scheduler, so typing:
 
 ::
 
-    (pangeo) $ qsub -I -l select=1:ncpus=4 -l walltime=03:00:00 -q regular
+    (pangeo) $ qsub -I -A account -l select=1:ncpus=4 -l walltime=03:00:00 -q regular
```

Comment:
-A account is required for Cheyenne, but not necessarily with other systems. This is not mandatory in CNES for example.

Reply:
It is indeed not required in many other systems, but at the beginning the page says the examples were developed using NCAR's Cheyenne and the sentence just before this says "In our case, the Cheyenne ... uses the PBS job scheduler, so typing:".

Reply:
OK, fine by me.